extensions/smallrye-graphql/runtime/src/main/java/io/quarkus/smallrye/graphql/runtime/SmallRyeGraphQLExecutionHandler.java

```diff
@@ -196,8 +196,8 @@ private JsonObject getJsonObjectFromBody(RoutingContext ctx) throws IOException
     }
 
     private String readBody(RoutingContext ctx) {
-        if (ctx.getBody() != null) {
-            return ctx.getBodyAsString();
+        if (ctx.body() != null) {
+            return ctx.body().asString();
```

Comment:
Next time, let's avoid to shoe in this type of change in a security fix as this is not backportable to 2.10.

Reply:
(I fixed it, no need for additional work on your side)